Sources/AWSSDKSwiftCore/XML.swift

```diff
@@ -0,0 +1,465 @@
+//
+//  XML.swift
+//  AWSSDKSwift
+//
+//  Created by Adam Fowler on 2019/06/15
+//
+//
+import Foundation
+
+public class XML {
```

Comment:
Did you read any docs and/or RFCs to help you implement this? If so, linking here would be awesome!

Also, if you can add some of the context you shared (issues with the version in Foundation, missing iOS support) here as well, that would help set future-readers down the right path.

Reply:
Done